src/main/java/gregtech/api/util/GTPP_Recipe.java

```diff
@@ -686,7 +686,7 @@ protected void drawNEIOverlayForInput(FixedPositionedStack stack) {
                 0,
                 0,
                 0,
-                0,
+                1,
```

Comment:
Also increase `mUsualInputCount` to make recipe search working

Reply:
We have the jar on a server and chemplant works. Should I increase it anyway?

Reply:
Item-only recipes won't work without it